Update docs_ubuntu_22.04.md (#649)
wireapp/wire-server-deploy: offline/docs_ubuntu_22.04.md

### Setting up Database network interfaces.
* Make sure that `assethost` is present in the inventory file with the correct `ansible_host` (and `ip` values if required)

> * Make sure that `cassandra_network_interface` is set to the private network interface on which

Reviewer: ```suggestion
* Make sure that `cassandra_network_interface` is set to the name of the network interface on which
```

>   the kubenodes can reach cassandra and on which the cassandra nodes

Reviewer: ```suggestion
  the kubenodes should talk to cassandra and on which the cassandra nodes
```

>   communicate among eachother. For example if you set up the kubenodes via virt-manager then the interface will be named `enp1s0`. Run `ip addr` on the kubenodes to determine the network interface.

Reviewer: ```suggestion
  communicate among each other. For example, if you set up the kubenodes via virt-manager then the interface will be named `enp1s0`. Run `ip addr` on the kubenodes to determine the network interface names. Select the interface that corresponds to the network you want cassandra and kubernetes to communicate over. 
```

>   communicate among each other. For example, if you set up the kubenodes via virt-manager then the interface will be named `enp1s0`. Run `ip addr` on the kubenodes to determine the network interface names. Select the interface that corresponds to the network you want cassandra and kubernetes to communicate over. 

Reviewer: ```suggestion
  should communicate among each other. Run `ip addr` on one of the cassandra nodes to determine the network interface names, and which networks they correspond to.  For example, if you set up the cassandra nodes via virt-manager, then the interface will be named `enp1s0`. 
```

* Similarly `elasticsearch_network_interface` and `minio_network_interface`

>   should be set to the same private network interface as well.

Reviewer: ```suggestion
  should be set to the network interface names you want elasticsearch and minio to communicate with kubernetes with, as well.
```
